packages/cra-template/scripts/test.ts

```diff
@@ -1,4 +1,4 @@
-import fs from 'fs-extra';
```

Comment:
while updating this file I took opportunity to get rid of unnecessary abstraction usage (fs-extra) as node ships those API natively